nmostesting/GenericTest.py

```diff
@@ -449,15 +468,16 @@ def do_test_api_resource(self, resource, response_code, api):
         # Gather IDs of sub-resources for testing of parameterised URLs...
         self.save_subresources(resource[0], response)
 
+        cors_valid, cors_message = self.check_CORS(resource[1]['method'], response.headers,
+                                                   cors_methods, cors_headers)
+        if not cors_valid:
+            # Fail for a plain CORS check
```

Comment:
What does 'plain' mean here and below? 😵 This check applies to all requests, the other to ones that don't need to check the body.